backend/project/views.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.views import APIView
```

Comment:
Works for now, but we'll probably want to move this out to a separate auth application in the project, rather than muddle up the base config folder.

Reply:
Yep, I plan on making changes to the code to organize it better.